inception-project/inception
# Refactoring/1060 refactor prediction task.java

**What's in the PR**
Refactoring changes made to make Prediction extraction code reusable.

**How to test manually**
* Start Inception
* Select a project
* Go to Annotations
* Annotations should appear and in the logs prediction information should appear.

**Automatic testing**
* existing unit tests for AL and Recommendation should pass.

**Documentation**
No.
